roles/kubespray-defaults/defaults/main/download.yml

```diff
@@ -131,27 +131,27 @@ skopeo_version: "v1.16.1"
 kube_major_version: "{{ kube_version | regex_replace('^v([0-9])+\\.([0-9]+)\\.[0-9]+', 'v\\1.\\2') }}"
 
 pod_infra_supported_versions:
+  v1.32: "3.10"
   v1.31: "3.10"
   v1.30: "3.9"
-  v1.29: "3.9"
 pod_infra_version: "{{ pod_infra_supported_versions[kube_major_version] }}"
 
 etcd_supported_versions:
+  v1.32: "v3.5.16"
```

Comment:
Should we upgrade to the latest version (v3.5.17)?

Reply:
The Kubernetes is using 3.5.16, https://github.com/kubernetes/kubernetes/blob/v1.32.0/go.mod#L66
So keep it. :-)

Reply:
Got it, thanks!